docs/UserGuide/Appendix/SQL-Reference.md

```diff
@@ -716,9 +716,27 @@ In this situation, it will throws an exception if * corresponds to multiple sens
 
 ```
 
+* Regexp Statement
+
+Regexp Statement only supports regular expressions with data type of text and Java standard library style when matching
+```
+SELECT <SelectClause> FROM <FromClause> WHERE  <WhereClause>
+Select Clause : <Path> [COMMA <Path>]*
+FromClause : < PrefixPath > [COMMA < PrefixPath >]*
+WhereClause : andExpression (OPERATOR_OR andExpression)*
+andExpression : predicate (OPERATOR_AND predicate)*
+predicate : (suffixPath | fullPath) LIKE stringLiteral
+stringLiteral : SINGLE_QUOTE_STRING_LITERAL | DOUBLE_QUOTE_STRING_LITERAL
+
+Eg. select s1 from root.sg.d1 where s1 regexp 'Regexp'
+Eg. select s1, s2 FROM root.sg.d1 where s1 regexp 'regexp' and s2 regexp 'Regexp'
+Eg. select * from root.sg.d1 where s1 regexp 'Regexp'
+Eg. select * from root.sg.d1 where s1 regexp 'Regexp' and time > 100
+```
+
 * Like Statement
 
-Fuzzy query only supports regular expressions with data type of text and Java standard library style when matching
+The usage of Like Statement similar with mysql, but only support the regular expressions with data type of text
```

Comment:
The usage of `LIKE` Statement is similar with mysql, but only support the regular expressions on time series  which is TEXT data type


BTW, I still do not get the difference between `regexp` and `like` in your description.